src/components/layout/user-menu.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Anchor from 'grommet/components/Anchor';
+import Menu from 'grommet/components/Menu';
+import LogoutButton from './logout-button';
```

Comment:
Is `LogoutButton` and `Anchor` still needed here since the component doesn't use them directly in the render?

Reply:
After reviewing this component a bit more (with another repo) it looks like these imports are being used because `user-menu` might be passed some Grommet components for the menu items. Disregard the previous comment if that understanding is correct.